extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -27,40 +32,31 @@ public function testLoadDataFromRev( $params ) {
 	}
 
 	/**
-	 * @param Revision $rev
 	 * @param array $params
-	 *
-	 * @return PHPUnit_Framework_MockObject_MockObject|WallNotificationEntity
 	 */
-	private function getEntityMock( Revision $rev, array $params ) {
-		$wm = $this->getWallMessageMock( $params );
-
-		$entity = $this->getMockBuilder( 'WallNotificationEntity' )
-			->setMethods( [
-				'getUserName',
-				'getWallMessageFromRev',
-			] )
-			->disableOriginalConstructor()
-			->getMock();
+	private function setupEnvironmentMocks( array $params ) {
+		$userMock = $this->getMock( User::class, [ 'getName' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getName" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)